jigar/hangman/hangman.rb

```diff
@@ -0,0 +1,45 @@
+$secret_word = "aaabbbcccc"
+$word = $secret_word.split("")
+$guess_word = " _" * $word.length
+$max_chances = 6
+$correct_guesses = 0
+
+def check_the_guess(letter)

```

Comment:
what if I guess same word again and again at one time my correct guess will be equals to word length and I will win instead of guessing complete word 
So i think this is not a correct logic